staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -84,7 +91,14 @@ func (a *AdmissionOptions) AddFlags(fs *pflag.FlagSet) {
 		"The names in the below list may represent a validating plugin, a mutating plugin, or both. "+
 		"Within each phase, the plugins will run in the order in which they are passed to this flag. "+
 		"Comma-delimited list of: "+strings.Join(a.Plugins.Registered(), ", ")+".")
-
+	fs.MarkDeprecated("admission-control", "Use --enable-admission-plugin or --disable-admission-plugin instead. Will be removed in a future version.")
+	fs.StringSliceVar(&a.EnableAdmissionPlugins, "enable-admission-plugin", a.EnableAdmissionPlugins, ""+
+		"admission plugins should be enabled in addition to default enabled ones. "+
+		"Comma-delimited list of admission plugins: "+strings.Join(a.Plugins.Registered(), ", ")+". "+
+		"The order of plugins in this flag does not matter.")
+	fs.StringSliceVar(&a.DisableAdmissionPlugins, "disable-admission-plugin", a.DisableAdmissionPlugins, ""+
+		"Comma-delimited list of admission plugins: "+strings.Join(a.Plugins.Registered(), ", ")+". "+
```

Comment:
admission plugins that should be disabled although they are in the default plugin list

Reply:
Thanks.